src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
-document.addEventListener('click', (e) => {
-  // write code here
+document.querySelector('.wall').addEventListener('click', (e) => {
+  const spider = document.querySelector('img.spider');
+
+  const div = e.currentTarget;
+  const rect = div.getBoundingClientRect();
+
+  const offsetX = e.clientX - rect.left;
+  const offsetY = e.clientY - rect.top;
+
+  const spiderWidth = spider.offsetWidth;
+  const spiderHeight = spider.offsetHeight;
+
+  let newLeft = offsetX - spiderWidth / 2 - 10;
```

Comment:
What is `2 - 10` here?
Do not use "magic" numbers - save them into variables with meaningful names